packages/data/src/payment-gateways/reducer.ts

```diff
@@ -7,8 +7,7 @@ import { Actions } from './actions';
 
 function updatePaymentGatewayList(
 	state: PluginsState,
-	paymentGateway: PaymentGateway,
-	selector: SelectorKeysWithActions
```

Comment:
It looks like `SelectorKeysWithActions` can be deleted from the import above, since it's no longer used.

Reply:
Good catch!  Updated!

Reply:
Did you push this commit?